modules/deploy/pages/deployment-option/self-hosted/manual/linux-system-tuning.adoc

```diff
@@ -0,0 +1,28 @@
+= Linux System Tuning
+:description: Learn how Redpanda applies automatic tunic to your Linux system.
+
+Redpanda includes numerous features to tune your Linux systems for optimal performance. You can trigger hardware-based optimizations through the `rpk redpanda tune` command, also called the autotuner. Software-based configurations are managed automatically through the `systemd` suite of tools.
```

Comment:
```suggestion
Redpanda includes several features to tune your Linux systems for optimal performance. You can trigger hardware-based optimizations through the `rpk redpanda tune` command, also called the autotuner. Software-based configurations are managed automatically through the `systemd` suite of tools.
```

Reply:
I only saw a few features mentioned--numerous suggests many more...hence, the wording update request here.